src/main/java/net/minestom/server/registry/Registry.java

```diff
@@ -410,6 +416,19 @@ public EnchantmentEntry(String namespace, Properties main, Properties custom) {
         }
     }
 
+    public record DamageTypeEntry(NamespaceID namespace, double exhaustion,
```

Comment:
exhaustion is a float

Reply:
can it cause issues when you convert it to NBT? currently the NBT conversion also uses double when it is actually a float